editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -3669,7 +3670,10 @@ void Node3DEditorViewport::focus_selection() {
 	cursor.pos = center;
 }
 
-void Node3DEditorViewport::assign_pending_data_pointers(Node3D *p_preview_node, AABB *p_preview_bounds, AcceptDialog *p_accept) {
+void Node3DEditorViewport::assign_pending_data_pointers(
+		Node3D *p_preview_node,
+		AABB *p_preview_bounds,
+		AcceptDialog *p_accept) {
```

Comment:
This looks like just a style change that isn't related to the rest of the changes, why is it in this PR?

Reply:
Ah, I think that was a holdover from an earlier revision where I was originally adding more parameters to the function. I'll change it back though since it's redundant now